release-notes/opensearch-ml-common.release-notes-2.16.0.0.md

```diff
@@ -0,0 +1,49 @@
+## Version 2.16.0.0 Release Notes
+
+Compatible with OpenSearch 2.16.0
+
+### Features
+* Add initial MLInferenceSearchResponseProcessor (#2688)[https://github.com/opensearch-project/ml-commons/pull/2688]
+* Add initial search request inference processor (#2731)[https://github.com/opensearch-project/ml-commons/pull/2731]
+* Feat: add bedrock runtime agent for knowledge base (#2651)[https://github.com/opensearch-project/ml-commons/pull/2651]
+
+### Enhancements
+* Automated model interface generation on aws llms (#2689)[https://github.com/opensearch-project/ml-commons/pull/2689]
+* Add Batch Prediction Mode in the Connector Framework for batch inference (#2661)[https://github.com/opensearch-project/ml-commons/pull/2661]
```

Comment:
This is a feature rather than an enhancement since the Model Predict API is changed. https://github.com/opensearch-project/ml-commons/issues/2488

Reply:
+1